.vscode/settings.json

```diff
@@ -6,5 +6,6 @@
     },
     "search.exclude": {
         "out": true // set this to false to include "out" folder in search results
-    }
+    },
+    "typescript.preferences.importModuleSpecifier": "relative"
```

Comment:
should we also do it for js?

    "javascript.preferences.importModuleSpecifier": "relative",

Reply:
Yeah might as well.